### PR TITLE
Label revamp

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

Update some labels thanks to @Milie44 !

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       |
| Added Behats                      |
| Changelog updated                 |
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) |
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

**Comment by damien-carcel** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 1]
```diff
@@ -341,23 +341,23 @@ pim_enrich:
...
                 locales:
                     title: Locales
-                    help: The locales you want to export
+                    help: The locales defines the localized product values to export. Ex:only product information in French.
                 attributes:
                     title: Attributes
                     edit: Edit
                     label: "{0} All attributes|{1} One attribute selected|]1,Inf[ {{ count }} attributes selected"
-                    help: The categories of the products you want to export
+                    help: Select the product information to export. Ex:only the technical attributes.
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: "Ex: only..." (missing space)

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 2]
```diff
@@ -341,23 +341,23 @@ pim_enrich:
...
                     title: Attributes
             filter:
                 channel:
                     title: Channel
-                    help: The channel you want to export
+                    help: The channel defines the scope for product values, the locales used to select data, and the tree used to select products.
                 locales:
                     title: Locales
-                    help: The locales you want to export
+                    help: The locales defines the localized product values to export. Ex:only product information in French.
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: "Ex: only..." (missing space)
 Add a line note

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 3]
```diff
@@ -372,7 +372,7 @@ pim_enrich:
                     empty_selection: All attributes will be exported
                 family:
                     title: Family
-                    help: The families you want to export
+                    help: Select the products to export by their family. Ex: Export only the shoes and dresses.
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: Same (and there is other ones)

**Comment by pierallard** [comment]
Gtm

**Pull request merged**